gamemodes/horde/gamemode/sh_horde.lua

```diff
@@ -60,6 +60,8 @@ CreateConVar("horde_enable_ammo_gui", 1, FCVAR_ARCHIVE, "Enables ammo UI.")
 CreateConVar("horde_enable_class_models", 1, FCVAR_ARCHIVE, "Enables ammo UI.")
 CreateClientConVar("horde_disable_default_gadget_use_key", 0, FCVAR_ARCHIVE, "Disable default key bind for active gadgets.")
 
+CreateClientConVar("horde_pickup_weapons", "1", true, true, "Allows the pickup of dropped weapons when walking over them.")
```

Comment:
This is running sv & cl, it should only run cl
put this and the convar above it in a if CLIENT block